src/index.html

```diff
@@ -0,0 +1,206 @@
+                aria-hidden="true"
+              >
+                <path
+                  fill-rule="evenodd"
+                  d="M12.707 5.293a1 1 0 010 1.414L9.414 10l3.293 3.293a1 1 0 01-1.414 1.414l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 0z"
+                  clip-rule="evenodd"
+                />
+              </svg>
+            </button>
+
+            <div
+              class="page relative inline-flex items-center border border-gray-300 bg-white text-sm font-medium text-gray-700 hover:bg-gray-50"
+            ></div>
+            <button
+              type="button"
+              class="next-btn relative inline-flex items-center px-2 py-2 rounded-r-md border border-gray-300 bg-white text-sm font-medium text-gray-500 hover:bg-gray-50"
+            >
+              <span class="sr-only">Next</span>
+              <!-- Heroicon name: solid/chevron-right -->
+              <svg
+                class="h-5 w-5"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 20 20"
+                fill="currentColor"
+                aria-hidden="true"
+              >
+                <path
+                  fill-rule="evenodd"
+                  d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                  clip-rule="evenodd"
+                />
+              </svg>
+            </button>
+          </nav>
+        </div>
+      </div>
+    </div>
+
+    <footer class="py-16 bg-white border-t border-stone-200">
+        <div class="px-8 max-w-7xl mx-auto">
+          <div class="grid grid-cols-3">
+            <div class="text-center">
+              <h2 class="text-sm font-bold text-stone-900">Company</h2>
+              <nav class="mt-6 grid gap-6 text-sm">
+                <a href="#">Join our Newsletter</a>
+                <a href="#">Advertise</a>
+                <a href="#">About Us</a>
+                <a href="#">Careers</a>
+                <a href="#">Contact Us</a>
+              </nav>
+            </div>
+            <div class="text-center">
+              <h2 class="text-sm font-bold text-stone-900">Social</h2>
+              <nav class="mt-6 grid gap-6 text-sm">
+                <a href="#">Facebook</a>
+                <a href="#">Twitter</a>
+                <a href="#">Instagram</a>
+                <a href="#">LinkedIn</a>
+              </nav>
+            </div>
+            <div class="text-center">
+              <h2 class="text-sm font-bold text-stone-900">Contact</h2>
+              <nav class="mt-6 grid gap-6 text-sm">
+                <a href="#">Email</a>
+                <a href="#">Phone</a>
+                <a href="#">Address</a>
+              </nav>
+            </div>
+          </div>
+          <div class="copyright">
+            <p class="text-sm text-center mt-6 text-gray-500">
+              &copy; 2022. Omar"Faruq" & Zuhra"Hashimi"
+            </p>
+          </div>
```

Comment:
- [ ] Kindly change the footer's inner text as you have to follow the required wireframe:
 
![image](https://user-images.githubusercontent.com/102612396/205019590-bbd1389b-33ad-47ee-bd80-5042fa081273.png)